yew/src/services/websocket.rs

```diff
@@ -400,6 +428,27 @@ mod tests {
         }
     }
 
+    #[test]
+    #[cfg(feature = "web_sys")]
+    async fn test_invalid_url_error() {
+        let url = "syntactically-invalid";
+        let cb_future = CallbackFuture::<Json<Result<Message, anyhow::Error>>>::default();
+        let callback = cb_future.clone().into();
+        let mut ws = WebSocketService::new();
+        let status_future = CallbackFuture::<WebSocketStatus>::default();
+        let notification: Callback<_> = status_future.clone().into();
+        let task = ws.connect_text(url, callback, notification);
+        assert!(task.is_err());
+        if let Err(task_err) = task {
+            if let WebSocketError::CreationError(some_error) = task_err {
```

Comment:
Can you please add an else branch so that we ensure that we are always checking the returned error?

Reply:
Yes.